src/backend/__tests__/utils.test.ts

```diff
@@ -88,79 +84,6 @@ describe('backend/utils.ts', () => {
     ).toEqual('fortnite')
   })
 
-  describe('getLatestReleases', () => {
```

Comment:
why are we removing this test if the function still exists in `src/backend/utils`?

Reply:
This test makes no sense for HyperPlay since we are not showing the latest release to the user anywhere in the App.
Heroic has it because when there is a new release we show it on the sidebar so the user can click on it and update.

So I could have updated it with HP info but since it doesn't test anything we use I just removed it since it is not useful now.

Reply:
should we remove the `getLatestReleases` function from utils also then?

Reply:
done